### ISSUE TITLE
New releases are not published to npm

### ISSUE THREAD

**Issue by ssbarnea** [redhat-developer/vscode-extension-tester#458]
I checked the tags and I seen that 4.4.0 was the latest, but I was surprised that my package manager was not finding a newer version, until I realised that https://www.npmjs.com/package/vscode-extension-tester lists 4.2.5 as the lastest version, and that is two months old and two significant releases older, which for a testing package is critical.

What is happening with the release pipeline?

**Comment by odockal** [comment]
@ssbarnea v4.4.0 tag was accidentally pushed but is not actual, I have removed it in order not to confuse users. v4.3.0 is in doing right now. 
As of 4.2.5 then, yes it is the latest release we did. And it is quite old for testing package. I agree.

**Comment by odockal** [comment]
@ssbarnea 4.3.0 is out: https://www.npmjs.com/package/vscode-extension-tester/v/4.3.0, hope that it will help you a bit before we will provide support for another versions.

**Comment by ssbarnea** [comment]
That reminds me of a recurring problem I had with extest. We test with one hardcoded oldest version, currently `1.48.0` and without latest (not mentioning the version with `-c`) which as of today appears to be `1.68.1`.

That means that extest will almost always download and use version of vscode that itself was never tested with, so we have no idea if something fails from that or from another cause.

I could prove very useful to have two placeholder versions that we can specify via `-c` that would select the oldest and newest versions that were used for testing extest itself, so we would have more confidence on the testing outcome.

That would also play really nice with updating versions with which we test with as they will be determined by the version of extest we have in our lock file, basically more control.

Basically i am proposing to expose tested/certified versions to the consumers. Probably this will work better even for extest own maintenance because you will not longer be forced to manually maintain GHA testing matrix. WYT?
